cmd/kube-apiserver/app/server.go

```diff
@@ -292,3 +294,9 @@ func Run(s *options.APIServer) error {
 	m.Run(s.ServerRunOptions)
 	return nil
 }
+
+func verifyEtcdServersList(options *genericoptions.ServerRunOptions) {
```

Comment:
Instead of copying this function into two places, this should probably be a public function in genericapiserver package.